internal/peer/node/start.go

```diff
@@ -204,7 +204,6 @@ func serve(args []string) error {
 	grpc.EnableTracing = true
 
 	logger.Infof("Starting %s", version.GetInfo())
-
```

Comment:
No need to touch this file :-)

Reply:
I'd updated this to include a log point to prove when testing that it was actually the peer I'd rebuilt.